src/deep-security/lib/CredentialsManager.js

```diff
@@ -147,10 +147,10 @@ export class CredentialsManager {
           return callback(true);
         });
       },
-      onDatasetDeleted: (dataset, datasetName, callback) => {
+      onDatasetDeleted: (dataset, datasetName) => {
```

Comment:
'datasetName' is defined but never used.<br>'dataset' is defined but never used.